### PR TITLE
Basic components

### PR DESCRIPTION
**Pull request by DeAlexandra**
Created:
-file structure;
-routing to: home, dashboard, transactions, payouts, users, error page;
-sidebar and the header part (logo missing);
-footer;

### CODE REVIEW

File: src/components/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap component="div" flexGrow={1}>
+            Lorem ipsum
+          </Typography>
+          <IconButton
+          position="right"
+            color="inherit"
+            // onClick={};
```
Comment (enfrbhd) [inline src/components/Sidebar.jsx:147]: please remove unnecessary comments

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                    mr: open ? 3 : "auto",
+                    justifyContent: "center"
+                  }}
+                >
+                  {icon}
+                </ListItemIcon>
+                <ListItemText primary={text} sx={{ opacity: open ? 1 : 0 }} />
+              </ListItemButton>
+            )
+                })}
```
Comment (enfrbhd) [inline src/components/Sidebar.jsx:190]: please use prettier or another code formatter to properly indent components

Comment (enfrbhd) [reply src/components/Sidebar.jsx:190]: use this code formatter to format all files as they don't have proper indentation

File: src/components/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          position="right"
+            color="inherit"
+            // onClick={};
+            edge="end"
+          >
+            <PersonIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+      <Drawer variant="permanent" open={open}>
```
Comment (enfrbhd) [inline src/components/Sidebar.jsx:154]: the components are becoming too big. Please separate the Drawer and List within other components

File: src/pages/users/Users.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Box } from '@mui/system'
+// import { Card, Typography, CardContent, CardMedia ,CardActionArea  } from '@material-ui/core'
```
Comment (enfrbhd) [inline src/pages/users/Users.js:3]: please remove unnecessary comments

File: src/components/Sidebar.jsx (hunk 1 above)
Comment (DeAlexandra) [reply src/components/Sidebar.jsx]: removed unnecessary comment

File: src/components/Sidebar.jsx (hunk 2 above)
Comment (DeAlexandra) [reply src/components/Sidebar.jsx]: used prettier formatter for indentation

File: src/pages/users/Users.js (hunk 4 above)
Comment (DeAlexandra) [reply src/pages/users/Users.js]: removed unnecessary comment

File: src/components/Sidebar.jsx (hunk 3 above)
Comment (DeAlexandra) [reply src/components/Sidebar.jsx]: split sidebar component into header, drawer and list components

**Review by enfrbhd — APPROVED** [review]

**Pull request merged**